src/current/v23.1/why-cockroachdb.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Why CockroachDB?
+summary: What is CockroachDB? How does it work? What makes it different from other databases?
+tags: postgres, cassandra, google cloud spanner
+toc: false
+docs_area: get_started
+---
+
+{% include {{ page.version.version }}/faq/what-is-crdb.md %}
+
+## Why use CockroachDB?
+
+There are many reasons to use CockroachDB, including:
+
+- [Resiliency](#resiliency)
+- [Scalability](#scalability)
+- [Strong consistency](#strong-consistency)
+- [Geo-partioning and multi-region features](#geo-partitioning-and-multi-region-features)
+- [PostgreSQL-compatibility](#postgresql-compatibility)
+
+### Resiliency
+
+One of the key attributes of CockroachDB is its inherent distributed nature. Data is automatically replicated across multiple nodes for high availability and failover protection, ensuring your data is always accessible. Even during instances of hardware failure or maintenance, the system remains resilient and operational.
+
+For more information, refer to [Fault Tolerance & Recovery](demo-fault-tolerance-and-recovery.html).
+
+### Scalability
+
+CockroachDB supports seamless and efficient horizontal scalability. As your data and transaction volumes increase, you can add more nodes to the cluster to manage this growth, all without a decline in system performance.
+
+For more information, refer to [Replication & Rebalancing](demo-replication-and-rebalancing.html).
+
+### Strong consistency
+
+CockroachDB supports [ACID](https://en.wikipedia.org/wiki/ACID) transactions. This means that every copy of data across all nodes accurately reflects the same state, thereby ensuring data integrity throughout your system.
+
+For more information, refer to [Transactions](transactions.html).
+
+### Geo-partitioning and multi-region features
```

Comment:
There should be body copy for this subsection.

Reply:
Added